docs/api-docs/provider-apis/tax/overview.md

```diff
@@ -78,21 +78,21 @@ Make sure to also review our [app development best practices](/api-docs/apps/gui
 
 ## Installing the app
 
-Once the merchant clicks **Install** on a tax providers single-click app, it's essential the app successfully handles the OAuth flow before moving onto the next step of establishing a connection with the [Tax Provider API](/api-reference/providers/tax-provider-api). This [OAuth summary](/api-docs/apps/guide/auth#oauth-summary) provides an overview of the OAuth flow.
+When the merchant clicks **Install** on a tax provider's single-click app, it's essential that the app successfully handle the OAuth flow before moving on to the next step of establishing a connection with the [Tax Provider API](/api-reference/providers/tax-provider-api). The [Apps Guide](/api-docs/apps/guide/auth) provides an overview of the OAuth flow.
```

Comment:
```suggestion
When the merchant clicks **Install** on a tax provider's single-click app, it's essential that the app successfully handles the OAuth flow before moving on to the next step of establishing a connection with the [Tax Provider API](/api-reference/providers/tax-provider-api). The [Apps Guide](/api-docs/apps/guide/auth) provides an overview of the OAuth flow.
```

Reply:
This clause is in the [subjunctive mood](https://www.thoughtco.com/subjunctive-mood-grammar-1692151). The verb should remain uninflected.